integr8ly/installation
# Add app metrics

## Additional Information

https://issues.jboss.org/browse/INTLY-2696

## Verification Steps

Run the installation playbook and when it is finished, you should see a project called `Mobile App Metrics` is created. There should be an operator running inside the project, along with the mobile app metric server.

NOTE:  the playbook may not run successfully due to an issue with the operator itself, it is being fixed in this [this PR](https://github.com/aerogear/app-metrics-operator/pull/7). But if you check the project and the services, they should be all up and running.

## Is an upgrade task required and are there additional steps needed to test this?

Run the upgrade playbook and you should see a project called `Mobile App Metrics` is created. There should be an operator running inside the project, along with the mobile app metric server.


TODO:
- [ ] Complete monitoring
